kuzzleio/sdk-javascript
# Migrate the CI (V7) to use KuzDoc

## What does this PR do?
Gets rid of `doc/doc.sh` and uses KuzDoc to build the docs.

### How should this be manually tested?

Go take a look at the CI job for the dead-link check.

To test the build and deploy, you can follow the steps of the [Doc Dev Deploy in the `.travis.yml` file](https://github.com/kuzzleio/sdk-javascript/pull/482/files#diff-354f30a63fb0907d4ad57269548329e3L114) and test them locally.
